tests/unit/versionUpdaterTest.js

```diff
@@ -0,0 +1,96 @@
+const {describe} = require('jest-circus');
+const functions = require('../../.github/libs/versionUpdater.js');
+
+const version = 'v2.3.9-80';
```

Comment:
Just FYI, we recently (the day before yesterday) got rid of the `v` prefix from all our versions and tags.

Reply:
Also, since this is a top-level constant it should be `UPPER_SNAKE_CASE`

```suggestion
const VERSION = 'v2.3.9-80';
```

Reply:
I am glad I've done this through a function! It is all done, I removed all 'v' from the front of the tags.